### PR TITLE
Minor tweak to the README.md file

### PR DESCRIPTION
**Pull request by Jainan-Tandel**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -40,7 +40,7 @@ Please see our [user guide](https://zcash.readthedocs.io/en/latest/rtd_pages/rtd
 
 ### Need Help?
 
-* :blue_book: See the documentation at the [ReadtheDocs](https://zcash.readthedocs.io)
+* :blue_book: See the documentation at the [ReadTheDocs](https://zcash.readthedocs.io)
```
[the remark this answers is not in the document]
Comment (Eirik0) [reply README.md]: I like this suggestion and is arguably preferable, but I am happy either way and would be OK with merging this as is. @str4d is your comment a blocking suggestion?

[the remark this answers is not in the document]
Comment (str4d) [reply README.md]: Non-blocking, the current change is still a readability improvement.

[the remark this answers is not in the document]
Comment (Eirik0) [reply README.md]: Ok I will just merge this as is and further improvements can be made later.

**Comment by Eirik0** [comment]
@zkbot r+

**Pull request merged**